salt/metalk8s/addons/prometheus-operator/deployed/storageclass.sls

```diff
@@ -1,14 +1,18 @@
-#!jinja | kubernetes kubeconfig=/etc/kubernetes/admin.conf&context=kubernetes-admin@kubernetes
+#! kubernetes kubeconfig=/etc/kubernetes/admin.conf&context=kubernetes-admin@kubernetes
```

Comment:
the move (and update) could have been in a dedicated commit, to explain the rationale behind the change/move.

Could also edit the current commit message to explain it.

Reply:
I'll amend the commit message. Not fond of doing moves like this (which don't bring any value per-se outside of the context of the actual commit doing the PV stuff and such) in separate commits.